src/window/mouse_manager.rs

```diff
@@ -232,26 +234,40 @@ impl MouseManager {
                         position: position.into(),
                         modifier_string: keyboard_manager.format_modifier_string("", true),
                     });
-                }
 
-                if down {
-                    self.dragging = Some(button_text);
+                    if down {
+                        self.drag_details = Some(DragDetails {
+                            button: button_text,
+                            draw_details: details.clone(),
+                        });
+                    } else {
+                        self.drag_details = None;
+                    }
                 } else {
-                    self.dragging = None;
+                    self.drag_details = None;
                 }
 
-                if self.dragging.is_none() {
+                if self.drag_details.is_none() {
                     self.has_moved = false;
                 }
             }
         }
     }
 
-    fn handle_line_scroll(&mut self, x: f32, y: f32, keyboard_manager: &KeyboardManager) {
+    fn handle_line_scroll(
+        &mut self,
+        x: f32,
+        y: f32,
+        keyboard_manager: &KeyboardManager,
+        renderer: &Renderer,
+    ) {
         if !self.enabled {
             return;
         }
 
+        let draw_details = self.get_window_details_under_mouse(renderer);
```

Comment:
draw details seems like it could have a more descriptive name. Maybe relevant_window_details? Or just window_details?

Reply:
I changed it to `window_details`